package.json

```diff
@@ -21,10 +22,12 @@
     "query-string": "^4.3.1",
     "react": "16.0.0-alpha.6",
     "react-native": "0.44.2",
+    "react-native-actionsheet": "^2.2.0",
     "react-native-communications": "^2.2.1",
     "react-native-device-info": "^0.10.2",
     "react-native-elements": "^0.9.3",
     "react-native-htmlview": "^0.9.0",
+    "react-native-material-design-searchbar": "^1.1.4",
```

Comment:
Any reason to add this dep in here? We already have a search bar in `react-native-elements` https://react-native-training.github.io/react-native-elements/API/searchbar/